### ISSUE TITLE
Unable to log in to WP admin on WP 5.7.1 PHP 8.0.3

### ISSUE THREAD

**Issue by xanni** [ciubotaru/share-on-diaspora#68]
```
PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($function) must be a valid callback, non-static method ShareOnDiaspora::diaspora_button_display() cannot be called statically in /var/www/katherinephelps/wp-includes/class-wp-hook.php:292
Stack trace:
#0 /var/www/katherinephelps/wp-includes/plugin.php(212): WP_Hook->apply_filters()
#1 /var/www/katherinephelps/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php(1768): apply_filters()
#2 /var/www/katherinephelps/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php(550): WP_REST_Posts_Controller->prepare_item_for_response()
#3 /var/www/katherinephelps/wp-includes/rest-api/class-wp-rest-server.php(1139): WP_REST_Posts_Controller->get_item()
#4 /var/www/katherinephelps/wp-includes/rest-api/class-wp-rest-server.php(985): WP_REST_Server->respond_to_request()
#5 /var/www/katherinephelps/wp-includes/rest-api.php(479): WP_REST_Server->dispatch()
#6 /var/www/katherinephelps/wp-includes/rest-api.php(2816): rest_do_request()
#7 [internal function]: rest_preload_api_request()
#8 /var/www/katherinephelps/wp-admin/edit-form-blocks.php(82): array_reduce()
#9 /var/www/katherinephelps/wp-admin/post-new.php(72): require('...')
#10 {main}
  thrown in /var/www/katherinephelps/wp-includes/class-wp-hook.php on line 292, referer: https://katherinephelps.com/wp-login.php?redirect_to=https%3A%2F%2Fkatherinephelps.com%2Fwp-admin%2Fpost-new.php&reauth=1
```

**Comment by ciubotaru** [comment]
Can you go to 'wp-content/plugins/share-on-diaspora/', find a file called 'class.php', go to line 139 and replace 'public function's with 'public static function'? Hopefully this should solve your issue.

**Comment by xanni** [comment]
Thanks, I'll try that!

**Comment by ciubotaru** [comment]
Please let me know how it goes, so I can release an update for everyone.

**Comment by xanni** [comment]
That seems to have resolved the issue.  Thanks!

**Comment by ciubotaru** [comment]
Great!

Fixed in 96e7ba089eeaa751ea7566dad712b5fa02a43c65. Closing.
